### ISSUE TITLE
Markdown-paper repo Issue

### ISSUE THREAD

**Issue by pylang** [tomduck/pandoc-fignos#49]
I've tried to use this [`markdown-paper` repo](https://github.com/ihrke/markdown-paper).  I came across this [issue](https://github.com/ihrke/markdown-paper/issues/1) with the figure numbers.  

Can you advise on what may be causing the error?

**Comment by bastionJersey** [comment]
Did you try replacing any reference links with full ones? That solved my problem here: https://github.com/tomduck/pandoc-fignos/issues/48 

It may not be the same issue at all, of course, but may be worth trying as it was during pre-processing that it failed to work for me.

**Comment by pylang** [comment]
@bastionJersey Thanks for the suggestion.  No I still get errors using a full path and reference paths (`./pics/dummy.pdf` and `/pics/dummy.pdf`).  I'd be grateful if anyone can confirm reproducing the described issue.

**Comment by tomduck** [comment]
Hi @pylang.  Thanks for your feedback.

Your link to the markdown-paper repo points back to this Issue.  I'm also not clear from the Issue what you are hoping I can look at.

Could you narrow down the problem and post some sample markdown that fails to be processed by pandoc-fignos?

Thanks,
Tom

**Comment by pylang** [comment]
@tomduck Thank you.  I fixed that link which just points to the project.  Please refer to item 3 of the [issue][1] regarding details.

---
> Footnotes and figure numbers give the following errors during compilation 

The output is a pdf file, but the figure numbers do not filter correctly.

---

You should be able to reproduce the error by cloning that repo and running `make`.  The required edits to the `Makefile` are noted in items 1 and 2.

1. replace `-S` with  `-f markdown+smart -t markdown-smart`.
1. in both line 8 and 9 substitute assignments with `./templates/dependencies/`

Run `make` and you get errors.  Among them are following:

![2018-08-07 18_04_39-make](https://user-images.githubusercontent.com/10778668/43805076-6da11630-9a6c-11e8-9f4b-3d25200c342b.png)

which outputs (`paper.pdf`)

![2018-08-07 18_02_37-paper pdf - foxit reader](https://user-images.githubusercontent.com/10778668/43805006-25c1fd34-9a6c-11e8-80e0-30c4eab0a37c.png)

where there should be a picture.  The specific markdown is in the `paper.md` file:

```
# Methods

Footnotes can be entered using this code[^1].

[^1]: a footnote

Figures are included like this.

![This is gonna be the caption.](pics/dummy.pdf){#fig:dummy}

And referenced from here as Fig. @fig:dummy.
```

Why don't the figures or figure numbers render correctly?  I'll try my best to assist with whatever questions you may have.  Many thanks.


  [1]: https://github.com/ihrke/markdown-paper/issues/1

**Comment by tomduck** [comment]
Hi @pylang.  I have it working.  However, my `Makefile` is a little different from yours.  In place of `-S` I used `-f markdown+smart`.  The `-t markdown-smart` flag forces the output to be markdown, which is not what you want.  I also ran `make -B` to force the `paper.tex` to be rebuilt.

Note that I used the most recent release of fignos/eqnos/tablenos, available as of today.  It is possible that cleaned up some other problems.
